CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Prowide ISO 20022 - CHANGELOG
 
+#### 9.3.5-SNAPSHOT
+  * Added DI support for JaxbContext
```

Comment:
DI? panel mas explicito/descriptivo... algo como "Added an optional way to pass a JaxbContext instance to the parse and write methods"

Reply:
Ok